src/test/java/com/example/DoughnutShopTest.java

```diff
@@ -0,0 +1,48 @@
+package com.example;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DoughnutShopTest {
+
+    @Test
+    public void testPrepareDoughnut_forChocolate() {
+        DoughnutShop shop = new DoughnutShop();
```

Comment:
Since we initialise `shop` in all our tests, can use @Setup that always initialises `shop` for all tests